src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -686,6 +688,10 @@ class HookUtils(object):
                 'name': 'execjob_resize',
                 'handler': self._execjob_resize_handler
             }
+            self.hook_events[pbs.EXECJOB_ABORT] = {
+                'name': 'execjob_abort',
+                'handler': self._execjob_abort_handler
```

Comment:
Suggest you call _execjob_end_handler() method for this.

Reply:
Good idea.